ams_flutter/lib/features/onboarding/presentation/pages/signup.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:ams_flutter/core/constants/app_colors.dart';
+import 'package:ams_flutter/core/constants/app_images.dart';
+import 'package:ams_flutter/core/constants/app_strings.dart';
+import 'package:ams_flutter/core/constants/app_text_styles.dart';
+import 'package:ams_flutter/features/onboarding/presentation/widgets/button.dart';
+import 'package:ams_flutter/features/onboarding/presentation/widgets/elevated_button.dart';
+import 'package:ams_flutter/features/onboarding/presentation/widgets/text_field_widget.dart';
+import 'package:flutter/material.dart';
+
+class SignUp extends StatefulWidget {
+  const SignUp({super.key});
+
+  @override
+  State<SignUp> createState() => _SignUpState();
+}
+
+class _SignUpState extends State<SignUp> {
+  TextEditingController _emailcontroller = TextEditingController();
```

Comment:
Could have used a `List<TextEditingController>` to reduce redundant variable declarations